### ISSUE TITLE
Profit/Loss & payout should take fees into account

### ISSUE THREAD

**Issue by da-kami** [itchysats/itchysats#1165]
Currently our profit/loss and payout display does not take fees into account in the UI.

Storing fees in the internal `Cfd` model is tracked in:  #1131 
This ticket should be tackled after or together with #1131 

The fee actually has to make sure to know it's sign internally and on the projection API, because fees for one party is a positive interest collection for the other. 

We have to keep track of the fees internally and use them in the profit calculation. The tests should be adapted to reflect the loss / gain in terms of fees for the parties. 
Once the projection calc for profit takes fees into account we should align the display in the UI and add a tooltip to the `payout` that makes it clear that `payout = initial_margin + profit - recurring_fees` (note: the opening fee is not part of this equation).

A note on calculating profit and loss based on what is outlined in #1164:
With reducing the margin on every rollover, our profit and loss will take recurring fees into account automatically. The payout curve will define the payout (taking all fees into account; recurring + opening). The profit and loss amount is then `p/l = margin + opening_fee - payout`.

**Comment by klochowicz** [comment]
Recording of fees is tracked in: #1131 , but you could plausibly say that the one you raised  is a follow-up issue that's blocked on the completion of it. Still, linking them is good as they're related :)
